.github/workflows/bundle.yml

```diff
@@ -22,18 +22,22 @@ jobs:
           pip install -r dev_requirements.txt
       - name: Bundle (MacOS)
         run: |
-          python -m nuitka --standalone --include-data-dir=source=source --noinclude-data-files='*.json' --remove-output --assume-yes-for-downloads --macos-create-app-bundle --macos-app-icon=source/resources/icon.png microcosm.py
+          python -m nuitka --standalone --include-data-dir=source=source --remove-output --assume-yes-for-downloads --macos-create-app-bundle --macos-app-icon=source/resources/icon.png microcosm.py
           ditto -c -k --sequesterRsrc --keepParent microcosm.app microcosm-osx.zip
         if: matrix.os == 'macos-latest'
       - name: Bundle (Ubuntu)
         run: |
-          python -m nuitka --onefile --standalone --include-data-dir=source=source --noinclude-data-files='*.json' --remove-output --assume-yes-for-downloads --linux-icon=source/resources/icon.png -o microcosm microcosm.py
+          python -m nuitka --onefile --standalone --include-data-dir=source=source --remove-output --assume-yes-for-downloads --linux-icon=source/resources/icon.png -o microcosm microcosm.py
           chmod 744 microcosm
           tar -czvf microcosm.tar.gz microcosm
         if: matrix.os == 'ubuntu-latest'
       - name: Bundle (Windows)
+        # The Windows installation of miniupnpc does not include the DLL that it requires to run. As such, we need to
+        # copy the miniupnpc DLL to the Python site-packages directory first, so that it can be picked up by Nuitka
+        # during the bundle.
         run: |
-          python -m nuitka --onefile --standalone --include-data-dir=source=source --noinclude-data-files='*.json' --remove-output --assume-yes-for-downloads --mingw64 --windows-icon-from-ico=source/resources/icon.png microcosm.py
+          cp source/resources/dll/miniupnpc.dll C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\
```

Comment:
It's probably dangerous to hard-code a patch version here when the action itself doesn't mention a patch version - so we'll have to add the patch version for the python version.